pkg/api/v1/atlascluster_types_test.go

```diff
@@ -0,0 +1,117 @@
+package v1
+
+import (
+	"reflect"
+	"sort"
+	"testing"
+
+	"github.com/fatih/structtag"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+var excludedClusterFieldsOurs = map[string]bool{}
```

Comment:
FYI: you can actually use empty `struct{}` as value type to avoid allocations (but please don't bother).

Reply:
ah, ok, great to know!